commons/src/main/java/io/ballerina/stdlib/graphql/commons/schema/types/DefaultDirective.java

```diff
@@ -16,7 +16,7 @@
  * under the License.
  */
 
-package io.ballerina.stdlib.graphql.commons.types;
+package io.ballerina.stdlib.graphql.commons.schema.types;
```

Comment:
Why are we changing the package name here?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-graphql/pull/1022/commits/89c318b365935d4112de3af29c687bda18da0707